src/libraries/System.Private.DataContractSerialization/src/System/Runtime/Serialization/DataContract.cs

```diff
@@ -2191,7 +2232,10 @@ internal static string SanitizeTypeName(string typeName)
             return typeName.Replace('.', '_');
         }
     }
+}
 
+namespace System.Runtime.Serialization
+{
```

Comment:
Mixing namespaces in a single file seem wrong. Should this code be moved to a separate file?

Reply:
I looked at the types left behind in the old namespace - they're all basically support types for the DataContract model. None of them are used in types outside of the DC object tree. So I just slurped them all into the new *.DataContracts namespace along with the type tree they support.